examples/plot_scalable_poly_kernels.py

```diff
@@ -0,0 +1,118 @@
+"""
+=======================================================
+Scalable learning with polynomial kernel aproximation
+=======================================================
+
+This example illustrates the use of :class:`PolynomialSampler` to efficiently
+generate polynomial kernel feature-space approximations. This is used
+to train linear classifiers that approximate the accuracy of kernelized ones.
+
+.. currentmodule:: sklearn.kernel_approximation
+
+We use the Covtype dataset [2]_, trying to reproduce the experiments on the
```

Comment:
The links `[1]_`and `[2]_` do not work for some reason, not sure if they work in examples.
Maybe just use `[1]` and `[2]`.

Reply:
done.